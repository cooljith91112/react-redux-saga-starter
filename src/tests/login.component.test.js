import React from 'react';

import renderer from 'react-test-renderer';
import LoginComponent from '../modules/login/login.component';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <LoginComponent />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});