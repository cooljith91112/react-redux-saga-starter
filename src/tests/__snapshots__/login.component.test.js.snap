// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
Array [
  <div
    className="col-md-12"
  >
    Login
  </div>,
  <div
    className="col-md-12"
  >
    <button
      className="btn btn-primary"
      onClick={[Function]}
    >
      Login
    </button>
  </div>,
]
`;
